### PR TITLE
Adding ansible file samples for snappi-sonic

### PR DESCRIPTION
**Pull request by selldinesh**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [ ] 202405

### CODE REVIEW

**Comment by selldinesh** [comment]
@sdszhang, added the snappi installation steps

**Comment by abdosi** [comment]
@selldinesh please updated testbed.yaml as csv is not being added.

File: ansible/files/sonic_example_ixia_link.csv [hunk 1]
```diff
@@ -1,4 +0,0 @@
-StartDevice,StartPort,EndDevice,EndPort,BandWidth,VlanID,VlanMode
-example-s6100-dut-1,Ethernet0,example-ixia-1,Card1/Port1,40000,,Access
-example-s6100-dut-1,Ethernet1,example-ixia-1,Card1/Port2,40000,,Access
-example-s6100-dut-1,Ethernet2,example-ixia-1,Card1/Port3,40000,,Access
```
Comment (rraghav-cisco) [inline ansible/files/sonic_example_ixia_link.csv:4]: @selldinesh , Is this format same for UHD100G32 as well ?

**Comment by selldinesh** [comment]
> @selldinesh please updated testbed.yaml as csv is not being added.

added the testbed.yaml file as discussed

File: ansible/files/sonic_example_ixia_link.csv (hunk 1 above)
Comment (kamalsahu0001) [reply ansible/files/sonic_example_ixia_link.csv:4]: you can specify as localuhd/18 or Card3/Port2 ... This is example for port 18 in UHD.

File: ansible/testbed.yaml [hunk 2]
```diff
@@ -212,3 +212,34 @@
...
+  topo: ptf64
+  ptf_image_name: docker-ptf-snappi
+  ptf: snappi-sonic-ptf
+  ptf_ip: 10.251.0.232
+  ptf_ipv6:
+  server: Server_6
+  vm_base:
+  dut:
+    - sonic-s6100-dut1
+  inv_name: lab
```
Comment (sdszhang) [inline ansible/testbed.yaml:227]: should be snappi-sonic?

File: ansible/testbed.yaml [hunk 3]
```diff
@@ -212,3 +212,34 @@
...
+  ptf_image_name: docker-ptf-snappi
+  ptf: snappi-sonic-ptf
+  ptf_ip: 10.251.0.232
+  ptf_ipv6:
+  server: Server_6
+  vm_base:
+  dut:
+    - sonic-s6100-dut1
+    - sonic-s6100-dut2
+  inv_name: lab
```
Comment (sdszhang) [inline ansible/testbed.yaml:243]: same here. snappi-sonic?

Comment (selldinesh) [reply ansible/testbed.yaml]: fixed

File: ansible/testbed.yaml (hunk 2 above)
Comment (selldinesh) [reply ansible/testbed.yaml]: fixed

**Review by sdszhang — APPROVED** [review]

**Review by abdosi — APPROVED** [review]

**Pull request merged**